src/PrestaShopBundle/Resources/views/Admin/Improve/International/Translations/translations_settings.html.twig

```diff
@@ -30,7 +30,10 @@
 
     {% block translations_kpis_row %}
         <div class="row">
-            {{ include('@PrestaShop/Admin/Common/Kpi/kpi_row.html.twig', {'kpiRow': kpiRow }) }}
+            {{ render(controller(
+                'PrestaShopBundle:Admin\\Common:renderKpiRow',
+                { 'factory': kpiRowFactory }
```

Comment:
@mickaelandrieu can I render it somehow without passing the factory to twig? :thinking:

Reply:
why not pass the Row instead of the factory?

Reply:
good idea :)